frontends/concrete-python/examples/tfhers-cml/test.py

```diff
@@ -0,0 +1,191 @@
+
+    circuit = compiler.compile(inputset_of_tfhe_rs_inputs)
+
+    tfhers_bridge = tfhers.new_bridge(circuit=circuit)
+    return circuit, tfhers_bridge
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command()
+@click.option("-s", "--secret-key", type=str, required=True)
+@click.option("-k", "--concrete-keyset-path", type=str, required=True)
+# This is where we generate the evaluation key at the Concrete format, from the
+# secret key coming from TFHE-rs, on the client side
+def keygen(secret_key: str, concrete_keyset_path: str):
+    """Concrete Key Generation"""
+
+    # Compile the Concrete function
+    circuit, tfhers_bridge = compile_concrete_function()
+
+    if os.path.exists(concrete_keyset_path):
+        os.remove(concrete_keyset_path)
+
+    # Load the initial secret key to use for keygen
+    with open(
+        secret_key,
+        "rb",
+    ) as f:
+        buff = f.read()
+
+    input_idx_to_key = {0: buff, 1: buff}
+    tfhers_bridge.keygen_with_initial_keys(input_idx_to_key_buffer=input_idx_to_key)
+
+    # FIXME: remove the secret key before saving. The secret key can be used for
+    # debugging but should really be removed in production
+    circuit.client.keys.save(concrete_keyset_path)
+
+
+@cli.command()
+@click.option("-c1", "--rust-ct-1", type=str, required=True)
+@click.option("-c2", "--rust-ct-2", type=str, required=True)
+@click.option("-o", "--output-rust-ct", type=str, required=True)
+@click.option("-k", "--concrete-keyset-path", type=str, required=True)
+# This is the actual FHE computation, on the server side
+def run(rust_ct_1: str, rust_ct_2: str, output_rust_ct: str, concrete_keyset_path: str):
+    """Run circuit"""
+    circuit, tfhers_bridge = compile_concrete_function()
+
+    if not os.path.exists(concrete_keyset_path):
+        raise RuntimeError("cannot find keys, you should run keygen before")
+
+    circuit.client.keys.load(concrete_keyset_path)
+
+    # read tfhers int from file
+    with open(rust_ct_1, "rb") as f:
+        buff = f.read()
+    # import fheuint8 and get its description
+    tfhers_uint8_x = tfhers_bridge.import_value(buff, input_idx=0)
+
+    # read tfhers int from file
+    with open(rust_ct_2, "rb") as f:
+        buff = f.read()
+    # import fheuint8 and get its description
+    tfhers_uint8_y = tfhers_bridge.import_value(buff, input_idx=1)
+
+    encrypted_x, encrypted_y = tfhers_uint8_x, tfhers_uint8_y
+
+    encrypted_result = circuit.run(encrypted_x, encrypted_y)
+
+    # export fheuint8
+    buff = tfhers_bridge.export_value(encrypted_result, output_idx=0)
+    # write it to file
+    with open(output_rust_ct, "wb") as f:
+        f.write(buff)
+
+    # BCM BEGIN: to debug computations
+    # FIXME: how does it decrypt? we are on the server side, we shouldn't have
+    # the secret key. I think it's because the secret key is saved in concrete_keyset_path
+
+    # x = circuit.decrypt(tfhers_uint8_x)
```

Comment:
this part shouldn't work, ie shouldn't be able to decrypt, but it does
I did https://github.com/zama-ai/concrete-internal/issues/874